### ISSUE TITLE
distracting keyboard view when using programmable keyboards

### ISSUE THREAD

**Issue by FraYoshi** [aradzie/keybr.com#63]
Having a programmable keyboard, I'm facing the issue to have a qwerty keyboard displayed on the training view while using a non-emulated dvorak... and it is distracting...

I propose to let the user to decide which layout to show, or to let the user disable the keyboard view.

**Comment by volking0** [comment]
There is a layout option in settings, it has dvorak and some more

**Comment by FraYoshi** [comment]
yes,
but when using a QMK programmable keyboard like the ZSA's Moonlander, the interface does not reflect the pressed keys...

I've uploaded a video to show the problem: https://www.youtube.com/watch?v=vQnSoyePOp0

**Comment by MrRTi** [comment]
In my experience clicking this button hides layout from screen, but the text is increased in size
<img width="299" alt="image" src="https://github.com/aradzie/keybr.com/assets/38328305/29e4ef2e-1ed1-4702-aca4-dd647ab535eb">

**Comment by FraYoshi** [comment]
oh.. never clicked it!
Thank you very much @MrRTi !
Still, having a definable custom layout would be top ^^

**Comment by sehrgut** [comment]
> Still, having a definable custom layout would be top

I agree! Parsing KLE layouts would probably be easiest both from a flexibility and an implementation standpoint, as the code to parse and render them in-browser already exists and is open-source.

**Comment by nebula-it** [comment]
To add to this, the issue is with the depressed key indicator. When emulation is turned off on non-qwerty layouts, the depressed key indicator still shows up at where the pressed key would have been on the QWERTY layout.
